### PR TITLE
Update Merkle tree example to make it work with 0.11.x

### PR DESCRIPTION
**Pull request by chiro-hiro**
I tried the Merkle tree example code but it wasn't work with lastest version `0.11.4` of SnarkyJS. I do my fix to make it work, hope it would help other people.

### CODE REVIEW

File: src/examples/zkapps/merkle_tree/merkle_zkapp.ts [hunk 1]
```diff
@@ -12,48 +12,41 @@ Merkle Trees give developers the power of storing large amounts of data off-chai
...
+class Account extends Struct({
+  publicKey: PublicKey,
+  points: UInt32,
+}) {
   hash(): Field {
-    return Poseidon.hash(this.toFields());
+    return Poseidon.hash([
+      ...this.publicKey.toFields(),
+      ...this.points.toFields(),
+    ]);
```
Comment (mitschabaude) [inline src/examples/zkapps/merkle_tree/merkle_zkapp.ts:42]: could we simplify this?

```suggestion
    return Poseidon.hash(Account.toFields(this));
```

**Review by mitschabaude — APPROVED** [review]
Thanks a lot for fixing the example!

File: src/examples/zkapps/merkle_tree/merkle_zkapp.ts (hunk 1 above)
Comment (chiro-hiro) [reply src/examples/zkapps/merkle_tree/merkle_zkapp.ts:42]: Wow, my bad. I don't know this exist.

File: src/examples/zkapps/merkle_tree/merkle_zkapp.ts [hunk 2]
```diff
@@ -12,48 +12,41 @@ Merkle Trees give developers the power of storing large amounts of data off-chai
...
   }
 
-  addPoints(n: number): Account {
-    return new Account(this.publicKey, this.points.add(n));
+  addPoints(points: UInt32) {
```
[the remark this answers is not in the document]
Comment (chiro-hiro) [reply src/examples/zkapps/merkle_tree/merkle_zkapp.ts:42]: I just want to keep it consistent with the struct but this one is unnecessary.

**Pull request merged**